src/qml/MessageBubble.qml

```diff
@@ -144,7 +144,7 @@ BorderImage {
             left: parent.left
             leftMargin: units.gu(1)
         }
-        width: paintedWidth > maxDelegateWidth ? maxDelegateWidth : undefined
+        //width: paintedWidth > maxDelegateWidth ? maxDelegateWidth : undefined
```

Comment:
This change is still rather warm. If you've confirmed that the issues fixed by https://github.com/ubports/messaging-app/commit/d644a2672bb333a5fad4d70a36949570d176eaff, please let me know.

You can delete this code, not just comment it out.

Reply:
I don't see any regressions with this commit